repl/src/main/scala/com/cloudera/livy/repl/PythonInterpreter.scala

```diff
@@ -34,11 +34,15 @@ import py4j.GatewayServer
 
 import com.cloudera.livy.Logging
 import com.cloudera.livy.client.common.ClientConf
+import com.cloudera.livy.sessions._
 
 // scalastyle:off println
 object PythonInterpreter extends Logging {
-  def apply(conf: SparkConf): Interpreter = {
-    val pythonExec = sys.env.getOrElse("PYSPARK_DRIVER_PYTHON", "python")
+  def apply(conf: SparkConf, kind: Kind = PySpark()): Interpreter = {
```

Comment:
Instead of a default value, I'd rather make this explicit when creating a `PythonInterpreter`.